cub/cub/device/dispatch/dispatch_scan.cuh

```diff
@@ -62,6 +63,46 @@
 
 CUB_NAMESPACE_BEGIN
 
+namespace detail
+{
+namespace scan
+{
+
+template <typename MaxPolicyT,
+          typename InputIteratorT,
+          typename OutputIteratorT,
+          typename ScanOpT,
+          typename InitValueT,
+          typename OffsetT,
+          typename AccumT,
+          bool ForceInclusive>
+struct DeviceScanKernelSource
+{
+  using ScanTileStateT = typename cub::ScanTileState<AccumT>;
+
+  CUB_DEFINE_KERNEL_GETTER(ScanInitKernel, DeviceScanInitKernel<ScanTileStateT>)
+
+  CUB_DEFINE_KERNEL_GETTER(
+    ScanKernel,
+    DeviceScanKernel<MaxPolicyT,
+                     InputIteratorT,
+                     OutputIteratorT,
+                     ScanTileStateT,
+                     ScanOpT,
+                     InitValueT,
+                     OffsetT,
+                     AccumT,
+                     ForceInclusive>)
+
+  CUB_RUNTIME_FUNCTION static constexpr std::size_t AccumSize()
```

Comment:
If NVRTC should be able to compile this, we need to use everything from `::cuda::std`
```suggestion
  CUB_RUNTIME_FUNCTION static constexpr ::cuda::std::size_t AccumSize()
```

Reply:
@griwes can correct me if I'm wrong, but it's not this header that needs to be NVRTC compilable, but rather just `device/dispatch/kernels/scan.cuh`. That header was made NVRTC compilable in https://github.com/NVIDIA/cccl/pull/3334 (also this is tested there)